File: 3st/Jaeyoung/최솟값 만들기.cs
using System;

public class Solution {
    public int solution(int[] A, int[] B) {
        int answer = 0;
        int sum = 0;
        Array.Sort(A);
        Array.Sort(B);
        for ( int i = 0, j = A.Length-1 ; i < A.Length ; i++, j-- ) {
            answer += A[i]*B[j];
        }
        return answer;
    }
}

File: 3st/Hyeonseo/programmer/최솟값 만들기.java
import java.util.Arrays;
import java.util.stream.IntStream;
class Solution{
    public int solution(int []A, int []B){
        Arrays.sort(A);
		Arrays.sort(B);
		int answer = 0;
		int[] c = IntStream.concat(Arrays.stream(A), Arrays.stream(B)).toArray();
		for(int i = 0; i < A.length; i++) {
			answer += c[i] * c[c.length - i - 1];
		}
        return answer;
    }
}

//level 2